tetris.py

```diff
@@ -132,6 +144,10 @@ def new_block(self):
             self.current_block = self.upcoming_block
         self.upcoming_block = random.choice(list(blocks.values()))
 
+    def square_color(self):
+        k = {(coord, random.choice(list(color_chart.values()))) for coord in self.landed_blocks}
+        print(k)
```

Comment:
How is this method supposed to work? It doesn't return anything, and it doesn't change any attribute of `self`.

Also, it makes a **set**, not a dict. Did you know this? Do you know what a set is?
```
{thing1: thing2 for something in something}    # makes a dict
{(thing1, thing2) for something in something}  # makes a set of tuples
```